README.md

```diff
@@ -156,8 +162,14 @@ Thanks!
 
 ## Upcoming Release
 * Added ElementalSkills option (+Cold,Lightning,Magic,Poison Skills)
```

Comment:
do you want to include this in the release? is there a UI option for it?

Reply:
There is now.

Reply:
this can be added to v0.5.4 in the readme